source/ptreferential/tests/ptref_ng_test.cpp

```diff
@@ -76,8 +76,8 @@ BOOST_AUTO_TEST_CASE(parse_pred) {
         R"#(vehicle_journey . has_code_type ( external_code ) )#",
         R"#(vehicle_journey.has_code_type("external_code"))#");
     assert_expr(
-        R"#(route . has_code_type ( forward ) )#",
-        R"#(route.has_code_type("forward"))#");
+        R"#(route . has_direction_type ( forward ) )#",
```

Comment:
because of late review : https://github.com/CanalTP/navitia/pull/2960#discussion_r328495559